private-docker-registry/README.md

```diff
@@ -0,0 +1,29 @@
+# Private Docker Registry
+
+This is a private docker registry for Dstack and Phala Cloud.
+
+The example of [docker-compose.yml](docker-compose.yml) is provided to help you to load docker images from a private docker registry.
+
+## Notices
+
+- The environment variables are required to be set in the Dstack and Phala Cloud through the `Encrypted Secrets` feature.
```

Comment:
is this example illustrating any features that are part of Phala Cloud and are not available/supported in the open source Dstack/teepod? I think the encrypted secrets and private registry both do not require phala Cloud specifically

Reply:
This example are both available for phala cloud and dstack.

Reply:
Essentially it's the same, because the cloud is just a managed Dstack deployment